### DISCUSSION TITLE
PK-PD modeling  using nlmixr2

### DISCUSSION THREAD

**Discussion by modeler466** [nlmixr2/nlmixr2#262] [General]
Hi guys,
I have worked on PK-PD modeling in commercial software which have a GUI, but i'm not familiar with modeling and simulation using nlmixr2.
I want to learn it, kindly help with understanding the same

Thanks in advance

**Comment by billdenney** [comment]
The best starting points are probably:

1. https://nlmixr2.org/articles/running_nlmixr.html
2. Reviewing some of the courses that have been given over time: https://github.com/nlmixr2/courses
